docs/rules/documentation.md

```diff
@@ -46,3 +46,29 @@ function in the model's `.yml` entry.
 !!! note "Tip"
 
     We recommend that every model in your dbt project has at minimum a model-level description. This ensures that each model's purpose is clear to other developers and stakeholders when viewing the dbt docs site. Missing documentation should be addressed first for marts models, then for the rest of your project, to ensure that stakeholders in the organization can understand the data which is surfaced to them.
+
+## Undocumented Source Nodes
```

Comment:
i think it would be good to include the little sample yml snippet you used in the conversation on this PR to clarify which description block would resolve each of these rules!

Reply:
Done!